x-pack/test/apm_api_integration/tests/transactions/transactions_groups_comparison_statistics.ts

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import url from 'url';
+import { APIReturnType } from '../../../../plugins/apm/public/services/rest/createCallApmApi';
+import archives from '../../common/fixtures/es_archiver/archives_metadata';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { registry } from '../../common/registry';
+import { removeEmptyCoordinates, roundNumber } from '../../utils';
+
+type TransactionsGroupsComparisonStatistics = APIReturnType<'GET /api/apm/services/{serviceName}/transactions/groups/comparison_statistics'>;
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+
+  const archiveName = 'apm_8.0.0';
+  const { start, end } = archives[archiveName];
+  const transactionNames = ['DispatcherServlet#doGet', 'APIRestController#customers'];
+
+  registry.when(
+    'Transaction groups agg results when data is not loaded',
```

Comment:
Would be great if this test suite's name was more reflective of what it's testing. IMHO the pathname of the endpoint is always a good one but 'Transaction groups comparision statistics' or something similar works as well.